corgi/web/templates/base.html

```diff
@@ -10,65 +10,45 @@
     <link rel="stylesheet" type="text/css" crossorigin="anonymous"
           href="https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.59.5/css/patternfly-additions.min.css"
           integrity="sha512-EYt56ZZC96V80F1+H7qwhi3NUIzR7onpd+RsVWHUxQfZSiIWmsQFD+6J/bS6OsAGKgvRftxtkxB16efK4mDFaA==">
+    <link rel="stylesheet" type="text/css" href="{% static 'base.css' %}">
+    <link rel="icon" href="{% static 'favicon.ico' %}">
     <title>Red Hat Component Registry | {% block title %}{% endblock %}</title>
+    <script defer src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"
+            integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg=="
```

Comment:
nit: I'm curious how these hashes are obtained or did you calculate them yourself?

Reply:
They were generated using https://www.srihash.org/